webapp/.gitignore

```diff
@@ -1,5 +1,6 @@
 .idea/*
 node_modules/*
+npm-debug.log
```

Comment:
move this to the gitignore in Lunchr (ie: one level up) and format as:

*/npm-debug.log

That way, the file is ignored across the entire project.